src/Supportive/Console/Subscriber/ConsoleSubscriber.php

```diff
@@ -50,10 +59,18 @@ public function onPreCreateAstMapEvent(PreCreateAstMapEvent $preCreateAstMapEven
         }
     }
 
+    /** @throws StopwatchException */
```

Comment:
I don't see how this exception is being handled in this PR.

Reply:
That is true, I did not know how to handle them within this subscriber. The exception will be thrown due to a bug (starting an already started event, or stopping a non-existing event), rather than a failure during the execution.

Since it is not a core functionality of Deptrac, I think it would be safe to just catch it in the `ConsoleSubscriber` and print `0` in case the `StopwatchException` is thrown.

How do you see it?

Reply:
I think, I would prefer recovering from the exception and displaying the old message here.

It's not like the user can do anything when something goes wrong and ignoring the timing will not affect Deptrac's analysis, so we can safely ignore it in case of errors.

Reply:
That is much better than what I proposed. I just applied the change. I had to do a bit of refactor in the `ConsoleSubscriber`, and to duplicate some test cases in order to cover all execution paths.